README.md

```
@@ -90,6 +90,8 @@ The arguments to `stemcell:build_with_local_os_image` are:
    of the most recent release and add one, e.g.: "0.0.7" → "0.0.8". If not
    specified, it will default to "0000".
 
+While building a stemcell for development purposes it is possible to set the BOSH_AGENT_BIN_PATH environment variable to inject an agent binary into the stemcell instead of downloading one. 
```

markdownlint-cli2 (0.22.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)